src/HttpClient.php

```diff
@@ -91,6 +93,60 @@ public function deploy(array $data)
         $this->guzzle->post('deploy', ['json' => $data]);
     }
 
+    /**
+     * Notify Bugsnag of a build.
+     *
+     * @param array $data the build information
+     *
+     * @return void
+     */
+    public function sendBuildReport(array $buildInfo)
+    {
+        $app = $this->config->getAppData();
+
+        $data = [];
+        $sourceControl = [];
+        if (isset($buildInfo['repository'])) {
+            $sourceControl['repository'] = $buildInfo['repository'];
+        }
+
+        if (isset($buildInfo['provider'])) {
+            $sourceControl['provider'] = $buildInfo['provider'];
+        }
+
+        if (isset($buildInfo['revision'])) {
+            $sourceControl['revision'] = $buildInfo['revision'];
+        }
+
+        if (!empty($sourceControl)) {
+            $data['sourceControl'] = $sourceControl;
+        }
+
+        if (isset($buildInfo['builder'])) {
+            $data['builderName'] = $buildInfo['builder'];
+        } else {
+            $data['builderName'] = exec('whoami');
```

Comment:
You should check if this function has been disabled, otherwise it'll crash. Many web hosts disable this by default for security.